workshops/FCR/README.md

```diff
@@ -0,0 +1,44 @@
+per image simultaneously in real time and display the probability of emotion in range of 0 to 1. Facial expression recognition work in 2 parts:
+1. Facial Detection:- The ability to detect the location of face in any input image or frame. The output is bounding box of the deteced faces.<br />
+2. Emotion Detection:- Classifying the emotion on the face as happy, angry, sad, surprise, disgust or fear.<br />
+
+Learn more about facial expression recognition in [Courcera course](https://www.coursera.org/projects/facial-expression-recognition-keras).<br/>
+[Link to DEMO](https://facialexpressionrecognition.yashkalbande.repl.co/) what we are going to build.
+
+## Lets Build the Model 
+Vist [teachablemachine.withgoogle.com](https://teachablemachine.withgoogle.com) and clicking on "Get Started". You will see three option to create an image, audio, or pose project. For now, pick “Image Project”.<br />
+![](https://cloud-kmaqeh4qc.vercel.app/create.gif)
+
+Click "Add a class" upto 6 classes. Rename "Class 1" to "Happiness" ,"Class 2" to "Sadness", "Class 3" to "Fear", "Class 4" to "Anger", "Class 5" to "Suprise" and "Class 6" to "Disgust"<br/>
+![](https://cloud-kmaqeh4qc.vercel.app/addclass.gif)
+Now is the most interesting step, we have to add image samples to each class with respective emotions. Like _smiling, happy_ images in __Happiness Class__. We have two option __Webcam__ and __Upload__ to add images. For video guide click [hear](https://youtu.be/DFBbSTvtpy4).<br />
+Click on __Train Model__ and let other values as default.Your Machine Learnig model is ready! Preview it by enabling webcam or input a image.Now its time to __Export Model__. We can export model in 3 differnt ways. But for now we will use default _Tensorflow.js_. Select __Upload my model__ to add our own model to cloud for further use.Click on _Copy_ option from code snippet. Open new HTML template on [repl.it](https://repl.it/languages/html) and paste the code between _body_ syntax.Click on __Run__ button to see your __Realtime Facial Expression Recognition__ working.Select __Start__ Button to get started. Give access to your webcam. It wil take couple of seconds to start webcam and connect to model.
+![](https://cloud-kmaqeh4qc.vercel.app/export.gif)<br />
+
+## Hacking
+There are different ways in which we can advance in Facial Expression Recognition. We can download Tensorflow model converted into keras .h5 model and load it in our Python Project.We get list for probablity of prediction for each class we add while creating Classes (Step: 4). Tensorflow Lite model to use in android application or IOT and   Take a look at classses we have add. We can add more class to it like Neautral, winking left or right side,etc. In the begining the result will be not that much accurate, but as time pass it will show more optimal results.Think of where we can use facial expression recoginition can be implement in real life. Below are some new example to keep hacking going on.<br />
```

Comment:
There are different ways in which we can advance in Facial Expression Recognition. We can download the Tensorflow model converted into Keras .h5 model and load it in our Python Project. We get a list for the probability of prediction for each class we add while creating Classes (Step: 4). Tensorflow Lite model to use in android application or IoT and Take a look at classes we have added. We can add more class to it like Neutral, winking left or right side, etc. In the beginning, the result will be not that much accurate, but as time pass it will show more optimal results. Think of where we can use facial expression recognition can be implemented in real life. Below are some new example to keep hacking going on.<br />